ansible_navigator/configuration_subsystem/navigator_post_processor.py

```diff
@@ -173,10 +174,48 @@ def mode(entry: Entry, config: ApplicationConfiguration) -> PostProcessorReturn:
         """Post process mode"""
         messages: List[LogMessage] = []
         errors: List[str] = []
-        subcommand = config.subcommand(config.app)
-        if entry.value.current not in subcommand.modes:
+        subcommand_action = None
+        subcommand_name = config.subcommand(config.app).name
+
+        for action_package_name in config.internals.action_packages:
+            try:
+                action_package = importlib.import_module(action_package_name)
+            except ImportError as exc:
+                message = f"Unable to load action package: '{action_package_name}': {str(exc)}"
+                messages.append(LogMessage(level=logging.ERROR, message=message))
+                continue
+            try:
+                if hasattr(action_package, "get"):
+                    subcommand_action = action_package.get(subcommand_name)  # type: ignore
+                break
+            except (AttributeError, ModuleNotFoundError) as exc:
+                message = f"Unable to load subcommand '{subcommand_name}' from"
+                message += f" action package: '{action_package_name}': {str(exc)}"
+                messages.append(LogMessage(level=logging.DEBUG, message=message))
+
+        if subcommand_action is None:
+            error = f"Unable to find an action for '{subcommand_name}', tried: "
+            error += oxfordcomma(config.internals.action_packages, "and")
+            errors.append(error)
+            return messages, errors
+
+        subcommand_modes = []
+
+        try:
+            getattr(subcommand_action, "run_stdout")
+            subcommand_modes.append("stdout")
+        except AttributeError:
+            pass
```

Comment:
```suggestion
        try:
            subcommand_action.run_stdout
        except AttributeError:
            pass
        else:
            subcommand_modes.append("stdout")
```

Thoughts on this instead as a minor style cleanup? It keeps the `try` block limited to its actual goal, and gets rid of an explicit call to `getattr()` since we know the attribute name upfront. Similar for below.

Reply:
think I'll leave the getattr, because it is explicit and doesn't annoy pylint.  adding the else, I always forget about that...